src/transformers/models/deberta/modeling_deberta.py

```diff
@@ -184,8 +184,8 @@ def __init__(self, config):
         self.attention_head_size = int(config.hidden_size / config.num_attention_heads)
         self.all_head_size = self.num_attention_heads * self.attention_head_size
         self.in_proj = nn.Linear(config.hidden_size, self.all_head_size * 3, bias=False)
-        self.q_bias = nn.Parameter(torch.zeros((self.all_head_size), dtype=torch.float))
-        self.v_bias = nn.Parameter(torch.zeros((self.all_head_size), dtype=torch.float))
+        self.q_bias = nn.Parameter(torch.zeros((self.all_head_size)), dtype=torch.float)  # Note: this is kept a float on purpose, even when the rest of the model is in bfloat.
```

Comment:
I am not sure the brackets here are correct.? `nn.Parameter` does not have a `dtype` argument, but `torch.zeros(...)` does.